deploy/aws-ecs/msdemo-cli

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
-PEM=microservices-demo-key
-STACK_NAME=microservices-demo-stack
-CF_FILE="file://$( cd $(dirname $0) ; pwd -P )/cloudformation.json"
+PEM=microservices-demo-key-jrs
```

Comment:
are these key name and stack name supposed to be like this? or you committed by mistake?